app/javascript/src/Policies/components/App.jsx

```diff
@@ -1,14 +1,19 @@
-/* eslint-disable import/no-named-as-default */
+// @flow
+
 import React from 'react'
 import PoliciesWidget from 'Policies/components/PoliciesWidget'
 
 // This is a class-based component because the current
 // version of hot reloading won't hot reload a stateless
 // component at the top-level.
 
-class App extends React.Component {
+type Props = {
+  store: any
```

Comment:
The thing with the store is that it is passed to redux directly, so typing it doesn't give us any benefit. And even we can't type any redux stuff because using generic types makes Jest fail.

Reply:
OK I managed to fix it in the end.